Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneEventCollector/TrendMicroVisionOneEventCollector.py

```diff
@@ -0,0 +1,1151 @@
+        log_id_field_name='uuid',
+        date_format=date_format,
+        latest_log_time=latest_log_time
+    )
+
+    return logs, dedup_log_ids, latest_log_time
+
+
+def get_workbench_logs(
+    client: Client,
+    first_fetch: str,
+    last_run: dict,
+    limit: int = DEFAULT_MAX_LIMIT,
+    date_format: str = DATE_FORMAT,
+) -> tuple[List[dict], dict]:
+    """
+    Get the workbench logs.
+
+    Args:
+        client (Client): the client object.
+        first_fetch (str): the first fetch time.
+        last_run (dict): the last run object
+        limit (int): the maximum number of workbench logs to return.
+        date_format (str): the date format.
+
+    Returns:
+        Tuple[List[Dict], Dict]: workbench logs & updated last run
+    """
+    def parse_workbench_logs(_workbench_logs):
+        for _log in _workbench_logs:
+            for _entity in (_log.get('impactScope') or {}).get('entities') or []:
+                if (related_entities := _entity.get('relatedEntities')) and isinstance(related_entities, list):
+                    _entity['relatedEntities'] = ','.join(related_entities)
+                if (provenance := _entity.get('provenance')) and isinstance(provenance, list):
+                    _entity['provenance'] = ','.join(provenance)
+                if (entity_value := _entity.get('entityValue')) and isinstance(entity_value, dict):
+                    if (_ips := entity_value.get('ips')) and isinstance(_ips, list):
+                        entity_value['ips'] = ','.join(_ips)
+
+    workbench_cache_time_field_name = LastRunTimeCacheTimeFieldNames.WORKBENCH.value
+    workbench_log_type = LogTypes.WORKBENCH.value
+    workbench_log_last_run_time = LastRunLogsStartTimeFields.WORKBENCH.value
+
+    start_time, end_time = get_datetime_range(
+        last_run_time=last_run.get(workbench_log_last_run_time),
+        first_fetch=first_fetch,
+        log_type_time_field_name=workbench_log_last_run_time,
+        date_format=date_format,
+    )
+
+    workbench_logs = client.get_workbench_logs(
+        start_datetime=start_time,
+        limit=limit + len(last_run.get(workbench_cache_time_field_name, []))
+    )
+
+    workbench_logs = dedup_fetched_logs(
+        logs=workbench_logs,
+        last_run=last_run,
+        log_id_field_name='id',
+        log_cache_last_run_name_field_name=workbench_cache_time_field_name,
+        log_type=workbench_log_type
+    )
+
+    latest_occurred_workbench_log_ids, latest_log_time = get_all_latest_logs_ids(
+        logs=workbench_logs,
+        log_type=workbench_log_type,
+        log_id_field_name='id',
+        date_format=DATE_FORMAT
+    )
+    parse_workbench_logs(workbench_logs)
+
+    latest_workbench_log_time = latest_log_time or end_time
+
+    workbench_updated_last_run = {
+        workbench_log_last_run_time: latest_workbench_log_time,
+        workbench_cache_time_field_name: latest_occurred_workbench_log_ids
+    }
+
+    demisto.info(f'{workbench_logs=}')
```

Comment:
we shouldnt print all this in info. lets switch to debug. last run should be in info

Reply:
see this: https://github.com/demisto/content/pull/26650/files#r1250301598